lit_gpt/config.py

```diff
@@ -1148,7 +1148,20 @@ def norm_class(self) -> Type:
         shared_attention_norm=True,
         lm_head_bias=True,
         gelu_approximate="tanh",
-    )
+    ),
+     dict(
```

Comment:
```suggestion
    # https://huggingface.co/microsoft/phi-2/blob/main/config.json
    dict(
```

Also looks like the dict is indented one extra space